appservice/src/intTest/groovy/nu/yona/server/CreateUserOnBuddyRequestTest.groovy

```diff
@@ -94,7 +92,7 @@ class CreateUserOnBuddyRequestTest extends AbstractAppServiceIntegrationTest
 		response.responseData.nickname == newNickname
 		response.responseData.devices.size() == 1
 		response.responseData.devices[0] == "iPhone 6"
-		response.responseData.goals.size() == 0 //TODO: updating of goals is not yet supported
+		//response.responseData._embedded._budgetGoals.size() == 0 //TODO: updating of goals is not yet supported
```

Comment:
Why is this commented? I would expect Bob to still have zero goals, so this line can be written as 
```
!response.responseData._embedded?._budgetGoals?.size
```